### PR TITLE
new autoupdate script using ansible

### PR DESCRIPTION
**Pull request by ozlevka-ericom**
https://github.com/EricomSoftwareLtd/SB/issues/2069

### CODE REVIEW

**Comment by ozlevka-ericom** [comment]
Add script take container version from version file

**Comment by ozlevka-ericom** [comment]
Hi @BenyH,
Please merge current PR...
I fix all spoken issues...

File: Setup/ericomshield-setup.sh [hunk 1]
```diff
@@ -518,6 +518,9 @@ function get_shield_files() {
     echo "Getting $ES_repo_restore_dev_sh"
     curl -s -S -o restore.sh "$ES_repo_restore_dev_sh"
     chmod +x restore.sh
+    echo "Getting $ES_repo_update"
+    curl -s -S -o restore.sh "$ES_repo_update"
```
Comment (BenyH) [inline Setup/ericomshield-setup.sh]: I assume this is a mistake, 
curl -s -S -o update.sh "$ES_repo_update"

File: Setup/shield-version.txt [hunk 2]
```diff
@@ -21,3 +21,4 @@ node-installer:latest node-installer:180214-12.26
 shield-logspout:latest shield-logspout:171123-17.23-642
 netdata:latest netdata:180114-08.17-1026
 speedtest:latest speedtest:180227-14.10-1464
+shield-autoupdate:latest shield-autoupdate:180314-15.28-1532
```
Comment (BenyH) [inline Setup/shield-version.txt]: Le't not change the prod version for now
(it will cause autoupdate)

File: Setup/autoupdate.sh [hunk 3]
```diff
@@ -91,17 +91,18 @@ while true; do
...
+                APPEND=""
+                if [ "$ES_DEV" == true ]; then
+                    APPEND="--dev"
+                fi
+
+                if [ "$ES_STAGING" == true ]; then
+                    APPEND="--staging"
+                fi
+
+                ./update.sh "$APPEND"
```
Comment (BenyH) [inline Setup/autoupdate.sh]: As discussed, let's comment these lines and use the old method until update.sh is tested (basic)

File: Setup/ericomshield-setup.sh (hunk 1 above)
Comment (ozlevka-ericom) [reply Setup/ericomshield-setup.sh]: Off course...

File: Setup/shield-version.txt (hunk 2 above)
Comment (ozlevka-ericom) [reply Setup/shield-version.txt]: OK

File: Setup/autoupdate.sh (hunk 3 above)
Comment (ozlevka-ericom) [reply Setup/autoupdate.sh]: OK...
I run interesting tests with new Broker. Might be I will commit this changes....

**Comment by ozlevka-ericom** [comment]
@BenyH 
Your changes is done

**Pull request merged**